apps/avatax/src/modules/avatax/calculate-taxes/avatax-calculate-taxes-adapter.ts

```diff
@@ -82,17 +79,24 @@ export class AvataxCalculateTaxesAdapter
     return normalizeAvaTaxError(err);
   }
 
-  async send(payload: CalculateTaxesPayload): Promise<AvataxCalculateTaxesResponse> {
+  async send(
+    payload: CalculateTaxesPayload,
+    config: AvataxConfig,
+    authData: AuthData,
+  ): Promise<AvataxCalculateTaxesResponse> {
```

Comment:
question: wouldn't be better to use an object instead of tree parameters?

Reply:
Yes but this is not in the scope of the refactor. I wanted only to extract services outside with minimal changes. Its hard with refactor, you change one thing and you introduce code changes for 100 files